contributors/devel/release/patch_release.md

```diff
@@ -0,0 +1,78 @@
+# Building Kubernetes patch releases
+
+This document describes the process for creating Kubernetes patch releases and communicating publishing
+status regarding known issues impacting the current release.
+
+## TL;DR
+
+The patch release czar will summarize all issues discovered since the last patch release weekly
+at the SIG release meeting and publish the summary to kubernetes-sig-release@googlegroups.com.
+
+## Process
+
+After a Kubernetes minor release, patch releases are managed by the Kubenretes patch release czar.
+The patch release czar works with SIGs to coordinate merging cherrypicks into the release branch,
+then cutting and publishing a new patch release.
+
+## Communicating critical fixes to the community
+
+In order to provide better transparency with the community on the patch release status and process,
+the patch release czar provides a weekly summary of all issues that have been discovered since the
+*last patch release* (for the most recent minor release only).
+
+The summary will be presented at the weekly sig-release meeting as the first agenda item (recorded
+in the release notes), and then mailed to kubernetes-sig-release@googlegroups.com.  During / after
+the summary at the SIG meeting, the patch release czar can answer questions from the community.
+
+The email update includes the state and a short summary for each item using the following template.
+(Description of fields provided in Parens)
+  
+```
+To: kubernetes-sig-release@googlegroups.com
+Subject: 1.7 PATCH RELEASE STATUS 6/28/17
+Body:
+
+#12345 (Issue number)
+- Status: Under development (Under investigation, Under development, In review, Merged)
+- Severity: Critical / Urgent (Critical / Urgent, Important Soon)
+- SIGs: Node, Networking (SIGs coordinating the fix)
+- Components: Kubelet, Kubeproxy (Impacted components)
+- Impact: foo can cause bar resulting in baz (Impact of the issue to the user or cluster)
+- PRs: #23456 (List of PR fixes)
+
+# 6789
+...
+```
+
+## Persistent record of issues discovered in each patch release
+
+The weekly summaries will be copied to the `release1.X/patch_history.md` document so it is easy
+to discover the history of all changes.
+
+```md
+# Kubernetes 1.X patch release history
+
+## 1.X.Y
+
+#12345 (Issue number)
+- Status: Under development (Under investigation, Under development, In review, Merged)
+- Severity: Critical / Urgent
+- SIGs: Node, Networking
+- Component: Kubelet
+- Impact: foo can cause bar
+- PRs: #23456
+- Resolution: 
+
+# 6789
+...
+
+## 1.X.(Y-1)
+
+...
+
+```
+
+## Communicating security fixes to the community
+
+Due to the sensitive nature of security fixes, they may be omitted from the weekly summary
```

Comment:
The patch release manager should probably mention in the sig-release meeting that they are working with the [product security team](https://github.com/kubernetes/community/blob/master/contributors/devel/security-release-process.md) on an issue, even if they can't say what the issue is.